dipy/align/meson.build

```diff
@@ -0,0 +1,54 @@
+cython_sources = ['bundlemin',
+'crosscorr',
+'expectmax',
+'parzenhist',
+'sumsqdiff',
+'transforms',
+'vector_fields',]
+
+cython_headers = [
+  'fused_types.pxd',
+  'transforms.pxd',
+  'vector_fields.pxd',
+]
+
+foreach ext: cython_sources
+  extra_args = []
+  # Todo: check why it is failing to compile with transforms.pxd
```

Comment:
Is this comment still relevant?

Reply:
Yes, still relevant but not a critic issue. I will dig into it later. 

it is just to detect change on `pxd` file and force a re-compilation. a feature that we did not have before